### DISCUSSION TITLE
n64 8bitdo kit button combo vibration

### DISCUSSION THREAD

**Discussion by supavgx** [darthcloud/BlueRetro#984] [Q&A]
As the title says sometimes while pressing key combo L+R+D-Right+C-Right the controller will vibrate  not everytime if i keep pressing the button combo sometimes it'll trigger a vibration sometimes it wont, does anyone know why this would be .

**Comment by darthcloud** [comment]
No idea , that's for the N64 digital OSD? There is some interferance produced by the OSD feature of HDMI board. Do you have the revised flex cable with the a buffer?

**Reply by supavgx** [reply]
Yes updated yea for n64d OSD and yes i have updated flex installed.
